### ISSUE TITLE
Use @sentry/react-native in Swift

### ISSUE THREAD

**Issue by hkhamm** [getsentry/sentry-react-native#1738]
### Environment

How do you use Sentry?
sentry.io

Which SDK and version?
"@sentry/react-native": "^2.6.2"

### Steps to Reproduce

1. `yarn add @sentry/react-native`
2. `cd ios; pod install`
3. In JS `Sentry.init(...)`
4. In a *.swift file in my native iOS code `import Sentry`
5. Attempt to build the app in Xcode

### Expected Result

Compiles successfully and starts creating issues in sentry.io

### Actual Result

Xcode reports a Swift Compiler Error > No such module 'Sentry'

<img width="1261" alt="Screen Shot 2021-08-20 at 1 03 16 PM" src="https://user-images.githubusercontent.com/1637732/130287674-fd08c20d-472b-4645-a9e5-c66b91f0d56b.png">

**Comment by hkhamm** [comment]
[This issue](https://github.com/getsentry/sentry-react-native/issues/813) suggests it is possible to `import Sentry` from Swift

**Comment by hkhamm** [comment]
Importing into `AppDelegate.m` with `#import <Sentry/Sentry.h>` works great.

I can `[SentrySDK captureMessage:@"Test message"];` without any issues.

**Comment by jennmueng** [comment]
@hkhamm Yes it is standard to use `import Sentry` in swift. Looks like you have something set up wrong for swift, maybe check out here: https://github.com/getsentry/sentry-cocoa/issues/362#issuecomment-572589461
